docs/guides/request_loaders.mdx

```diff
@@ -0,0 +1,126 @@
+
+Here is a basic example of working with the <ApiLink to="class/RequestList">`RequestList`</ApiLink>:
+
+<CodeBlock className="language-python">
+    {RlBasicExample}
+</CodeBlock>
+
+## Request manager
+
+The <ApiLink to="class/RequestManager">`RequestManager`</ApiLink> extends `RequestLoader` with write capabilities. In addition to reading requests, a request manager can add or reclaim them. This is important for dynamic crawling projects, where new URLs may emerge during the crawl process. Or when certain requests may failed and need to be retried. For more details refer to the <ApiLink to="class/RequestManager">`RequestManager`</ApiLink> API reference.
+
+## Request manager tandem
+
+The <ApiLink to="class/RequestManagerTandem">`RequestManagerTandem`</ApiLink> class allows you to combine the read-only capabilities `RequestLoader` (like <ApiLink to="class/RequestList">`RequestList`</ApiLink>) with a writable capabilities `RequestManager` (like <ApiLink to="class/RequestQueue">`RequestQueue`</ApiLink>). This is useful for scenarios where you need to load initial requests from a static source (like a file or database) and dynamically add or retry requests during the crawl. Additionally, it provides deduplication capabilities, ensuring that requests are not processed multiple times. Under the hood, <ApiLink to="class/RequestManagerTandem">`RequestManagerTandem`</ApiLink> manages whether the read-only loader still has pending requests. If so, each new request from the loader is transferred to the manager. Any newly added or reclaimed requests go directly to the manager side.
+
+### Request list with request queue
+
+This sections describes the combination of the <ApiLink to="class/RequestList">`RequestList`</ApiLink> and <ApiLink to="class/RequestQueue">`RequestQueue`</ApiLink> classes. This setup is particularly useful when you have a static list of URLs that you want to crawl, but you also need to handle dynamic requests during the crawl process. The <ApiLink to="class/RequestManagerTandem">`RequestManagerTandem`</ApiLink> class facilitates this combination, with the <ApiLink to="class/RequestLoader#to_tandem">`RequestLoader.to_tandem`</ApiLink> method available as a convenient shortcut. Requests from the <ApiLink to="class/RequestList">`RequestList`</ApiLink> are processed first by enqueuing them into the default <ApiLink to="class/RequestQueue">`RequestQueue`</ApiLink>, which handles persistence and retries for failed requests.
+
+<Tabs groupId="request_manager_tandem">
+    <TabItem value="request_manager_tandem_explicit" label="Explicitly usage">
+        <CodeBlock className="language-python">
+            {ExplicitTandemExample}
+        </CodeBlock>
+    </TabItem>
+    <TabItem value="request_manager_tandem_helper" label="Using to_tandem helper" default>
+        <CodeBlock className="language-python">
+            {TandemExample}
+        </CodeBlock>
+    </TabItem>
+</Tabs>
+
+## Conclusion
```

Comment:
Not sure where, but we should mention that you can pop just about any async generator into the `RequestList` and it can save you some memory when running locally.